pkg_defs/find_attitude/meta.yaml

```diff
@@ -3,6 +3,7 @@ package:
   version:  {{ SKA_PKG_VERSION }}
 
 build:
+  number: 1
```

Comment:
@taldcroft do you want to remove find_attitude from ska3-matlab or just remove the sherpa dependence?  Don't have a 32bit sherpa

Reply:
Remove find_attitude.  We can put it back when we don't support 32-bit.  I'd like to be able to call find_attitude from MATLAB tools using MAUDE telemetry ala Mark's attitude finder.  But not for this release!

Reply:
But to be clear, find_attitude does have a hard requirement for sherpa and it is useless without it.

Reply:
Yes, as far as I could tell... by removing the dependence from the meta you just allow the package to be installed.  So you'd end up with find_attitude installed and non-functional on 32bit,and find_attitude installed and functional on 64bit.

So this is just a lazy hack to avoid needing to put find_attitude into ska3-core (arch specific) and still gives you find_attitude on 64 bit.

But of course also fine to just pull it from ska3-matlab if you don't really care.

Reply:
The truth is that I don't really care either way.  But it just occurred to me that kadi 5.0 might be unhappy without find_attitude at least installed?  Find_attitude will definitely import without sherpa, so maybe that is good enough.  So maybe the recipe hack is the right way?

Reply:
I built with this recipe hack (package presently in the test channel)  and will just test it then and come back to this if it broke anything.